jekyll/_config.yml

```diff
@@ -154,6 +154,7 @@ defaults:
       suggested: false # Disable helpful resources section by default
       sitemap: true # Enable sitemap by default
       readtime: true # Enable Read Time by default
+      tags: null
```

Comment:
Should we set tags default null? To always have tags be defined? Or have it as an optional variable @halie-kastl 

Version use to always be defined in src/utils/types.ts, version: string[]; but had no default state.

Should we not have a default state for tags and keep it optional? 
Or have a default state and keep it defined?

Reply:
This is the state I mentioned that we should verify works. We will need to make sure this is compatible all the way through re-indexing. 

If possible, I'd like it to always we present. this will make future experiments more reliable, and it will help the docs team know where to place the tags they create. The same pattern should be followed through to section tags

Reply:
if the field is on some meta-data and not others, we could run into more issues than an occasional null state.